atlasexec/atlas.go

```diff
@@ -63,13 +63,20 @@ type (
 		RevisionsSchema string
 		Vars            Vars
 	}
+	ContextInput struct {
```

Comment:
Add godoc

Reply:
The `Input` suffix is a  GraphQL thing in Atlas. Here, it can be simply by `RunContext` (or `LintContext` and `PushContext` - another typedef).